baseapp/options.go

```diff
@@ -0,0 +1,28 @@
+package baseapp
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// File for storing in-package BaseApp optional functions,
```

Comment:
Cool - is it also possible to set these options in `config.toml` or a Gaia-specific file / is there some way we could architect that?

Reply:
We should definitely have this able to be set in config, but I think currently everything in the toml is managed by tendermint, so we would either have to add it there, or make a another config file. I would say we should make a separate issue to plan out how we want to do gaia/app specific config files.

Reply:
https://github.com/cosmos/cosmos-sdk/issues/1662